packages/eslint-plugin/src/rules/no-deprecated.ts

```diff
@@ -271,11 +271,32 @@ export default createRule({
       );
     }
 
+    function getJSXAttributeDeprecation(
+      openingElement: TSESTree.JSXOpeningElement,
+      node: IdentifierLike,
+    ): string | undefined {
+      if (openingElement.name.type !== AST_NODE_TYPES.JSXIdentifier) {
```

Comment:
Why this? Do we not want to check `<Tab.Item>`?

Reply:
I didn't consider the cases where <Tab.Item> is used. Thank you for pointing that out!